core/src/avm2/globals/flash/events/IOErrorEvent.as

```diff
@@ -1,7 +1,28 @@
 package flash.events {
     public class IOErrorEvent extends ErrorEvent {
+        // Defines the value of the `type` property of an `ioError` event object.
         public static const IO_ERROR:String = "ioError";
 
+        // These next three are undocumented, and are not supposed to be inspectable.
```

Comment:
Non-blocking, but is this comment a bit outdated since the Inspectable metadata was removed?

Reply:
I added this comment when I removed the metadata...
Oh, I have just now realized that the "... is metadata that the Flash IDE uses" comment was not just about it the metadata not being used by Flash Player, but also that it's entire intention doesn't make sense outside of the IDE. So I'll remove that part. Thanks!